### PR TITLE
python: when enabling uv and venv, version is not respected

### PR DESCRIPTION
**Pull request by gotcha**
@domenkozar This does work for me.

However, there are no tests.

Further, being a devenv newbie, I have no idea how many best practices I am breaking.

### CODE REVIEW

File: src/modules/languages/python.nix [hunk 1]
```diff
@@ -62,8 +62,8 @@ let
         [ -f "${config.env.DEVENV_STATE}/poetry.lock.checksum" ] && rm ${config.env.DEVENV_STATE}/poetry.lock.checksum
       ''}
       ${if cfg.uv.enable then ''
-        echo uv venv "$VENV_PATH"
-        uv venv "$VENV_PATH"
+        echo uv venv -p ${cfg.package.interpreter} "$VENV_PATH"
```
Comment (domenkozar) [inline src/modules/languages/python.nix:65]: Should it be package.interpeter?

File: src/modules/languages/python.nix [hunk 2]
```diff
@@ -48,7 +48,7 @@ let
 
     VENV_PATH="${config.env.DEVENV_STATE}/venv"
 
-    profile_python="$(${readlink} ${package.interpreter})"
+    profile_python="$(${readlink} ${cfg.package.interpreter})"
```
Comment (domenkozar) [inline src/modules/languages/python.nix:51]: Why is this needed?

**Review by domenkozar — CHANGES_REQUESTED** [review]

File: src/modules/languages/python.nix (hunk 1 above)
Comment (gotcha) [reply src/modules/languages/python.nix]: Thanks for the pointer. And yes, it should be ``package.interpreter``

File: src/modules/languages/python.nix (hunk 2 above)
Comment (gotcha) [reply src/modules/languages/python.nix]: Right, this is useless.

**Comment by gotcha** [comment]
@domenkozar Thanks for merging.

Where should I look if I would want to add a test ?

**Comment by domenkozar** [comment]
tests folder
